projects/uitpas/reference/uitpas.json

```diff
@@ -169,7 +169,7 @@
             }
           },
           "400": {
-            "description": "Bad Request. The response will include a JSON body with more details in the form of a RFC7807 error model. Possible error types:\n\n* https://api.publiq.be/probs/uitpas/event-not-found\n* https://api.publiq.be/probs/uitpas/passholder-not-found\n* https://api.publiq.be/probs/uitpas/passholder-no-active-cardsystems\n* https://api.publiq.be/probs/uitpas/social-tariff-expired\n* https://api.publiq.be/probs/uitpas/invalid-uitpasnumber\n* https://api.publiq.be/probs/uitpas/invalid-card\n* https://api.publiq.be/probs/uitpas/invalid-card-status\n\n",
+            "description": "Bad Request. The response will include a JSON body with more details in the form of a RFC7807 error model. Possible error types:\n\n* https://api.publiq.be/probs/uitpas/event-not-found\n* https://api.publiq.be/probs/uitpas/passholder-not-found\n* https://api.publiq.be/probs/uitpas/passholder-no-active-cardsystems\n* https://api.publiq.be/probs/uitpas/social-tariff-expired\n* https://api.publiq.be/probs/uitpas/social-tariff-suspended\n* https://api.publiq.be/probs/uitpas/invalid-uitpasnumber\n* https://api.publiq.be/probs/uitpas/invalid-card\n* https://api.publiq.be/probs/uitpas/invalid-card-status\n\n",
```

Comment:
Deze error-links zijn redirects en landen op de algemene pagina (bovenaan).
Is het niet logischer om de effectieve links te gebruiken?
Die hebben de volgende vorm:
* https://docs.publiq.be/docs/uitpas/errors#passholder-no-active-cardsystems
* https://docs.publiq.be/docs/uitpas/errors#social-tariff-expired
* https://docs.publiq.be/docs/uitpas/errors#social-tariff-suspended
* https://docs.publiq.be/docs/uitpas/errors#invalid-uitpasnumber
* https://docs.publiq.be/docs/uitpas/errors#invalid-card
* https://docs.publiq.be/docs/uitpas/errors#invalid-card-status

Mogelijk is er een reden om de API-urls te verkiezen - maar deze landen niet op de juiste anchor op de pagina.

Reply:
UiTPAS errors zitten momenteel onder UiTPAS in een subheader "errors" en niet onder het hoofd-item "Errors". Is het een groot werk om dit consistent te maken?